mappings/net/minecraft/client/render/block/entity/HangingSignBlockEntityRenderer.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/unmapped/C_zcgkklbo net/minecraft/client/render/block/entity/HangingSignBlockEntityRenderer
+	FIELD f_gzejleyy PLANK Ljava/lang/String;
+	FIELD f_kydnbzbo CHAIN_R2 Ljava/lang/String;
+	FIELD f_muvciuny NORMAL_CHAINS Ljava/lang/String;
+	FIELD f_ncuabxhv MODELS Ljava/util/Map;
+	FIELD f_nxjfptjq CHAIN_R1 Ljava/lang/String;
+	FIELD f_qicbzikq V_CHAINS Ljava/lang/String;
```

Comment:
What is this?

Reply:
its the string name for the model part. I just copied them